test/index.js

```diff
@@ -91,7 +91,7 @@ describe('Likely', function () {
         it('should change the shared URL when `<link rel="canonical">` is specified', function () {
             return driver.executeScript(`
                 document.head.innerHTML += '<link rel="canonical" href="https://google.com">';
-                likely.initiate();
+                likely.default.initiate();
```

Comment:
Woah, this is something we shouldn’t change. `likely.initiate()` is a public API, and this is an undesirable breaking change. I think we can handle it with e.g. [`babel-plugin-add-module-exports`](https://www.npmjs.com/package/babel-plugin-add-module-exports).